traffic_ops_db/test/docker/run-db-test.sh

```diff
@@ -99,12 +99,6 @@ fi
 
 ./db/admin --env=production upgrade || { echo "DB upgrade failed!"; exit 1; }
 
-if ! ./db/admin -env=production load_schema ||
-  ! ./db/admin -env=production load_schema; then
-  echo 'Could not re-run create_tables.sql!'
-  exit 1
-fi;
-
```

Comment:
As pointed out to me by @ocket8888, this means #4984 becomes reproducible again, which would be fixed by fixing #6106.

Reply:
I'm good with that, if you are.

Reply:
Yep that's fine with me, as long as #6106 gets fixed at some point.